### ISSUE TITLE
"unwrap.h" "timing.h" "SkinProbablilityMaps.h" Not found

### ISSUE THREAD

**Issue by hayalk** [headupinclouds/cvsteer#1]
"unwrap.h" "timing.h" "SkinProbablilityMaps.h" the three files are missing.

**Comment by headupinclouds** [comment]
I'll clean this up a little sometime later this week.

**Comment by headupinclouds** [comment]
@hayalk : This was a sandbox from a few years back.  I've removed unrelated code, updated CMake (package config install), added Hunter package manager for the OpenCV dependencies, and included a travis build test (building....).  Will add appveyor tests for Windows later.  Let me know if this fixes things for you.

**Comment by headupinclouds** [comment]
@hayalk: The travis builds are now passing on all platforms (iOS, Android, OS X, Linux).  Let me know if you have any problems.  Closing.

**Comment by hayalk** [comment]
Compiles without problems but the 

std::string filename = parser.getstd::string("input");
std::string output = parser.getstd::string("output");

in steer.cpp gets "true" no matter what I put for - input or -output

**Comment by headupinclouds** [comment]
@hayalk :  I've just tested and can't reproduce the problem.  Are you building with a local OpenCV version that is < 3.0.0 by any chance?  There was a change in the command line parser specification string that occurred in that version jump.  I've added a potential (untested) fix for builds using OpenCV < 3.0.0.  See `CV_DELIMITER` in the parser table:

https://github.com/headupinclouds/cvsteer/blob/master/test/steer.cpp#L199-L217

I hope that helps.  Feel free to send a pull request if you have any fixes.
